### PR TITLE
112 group things

### PR DESCRIPTION
**Pull request by cdmo**
Issue #112 

calling this "Group things" was a mistake, really it's about "flattening." 

Additional change: updated rubocop settings

### CODE REVIEW

File: app/services/place_hold_form/builder.rb [hunk 1]
```diff
@@ -44,10 +44,24 @@ def parsed_symphony_response(symphony_call, *params)
...
       @volumetric_calls = @call_list.dup if volumetric? && @call_list.count > 1
       volumetric_natural_sort
+      compact_non_volumetric_calls if @volumetric_calls.select { |c| c.volumetric.nil? }.count > 1
+      @volumetric_calls.uniq!(&:call_number)
+    end
+
+    def compact_non_volumetric_calls
+      process_these = @volumetric_calls.dup
+      @volumetric_calls = []
+      process_these.each_with_index do |c, i|
```
Comment (banukutlu) [inline app/services/place_hold_form/builder.rb]: a minor thing but per our conversation, renaming `c`  to be more descriptive would be helpful here

Comment (cdmo) [reply app/services/place_hold_form/builder.rb]: rename `c` or `process_these`?

i use `c` through the class as shorthand for `call`... 🤔 

`process_these` was a var i just forgot to rename before PRing... maybe I should rename both?

Comment (banukutlu) [reply app/services/place_hold_form/builder.rb]: oh I think `process_these` is fine, sorry I just meant `c`. I mean if you want to rename the `process_these` as well, your call but it is clear to read to me as is :)

Comment (cdmo) [reply app/services/place_hold_form/builder.rb]: okie doke, hope that improves readability

Comment (banukutlu) [reply app/services/place_hold_form/builder.rb]: looks great, thanks for the updates!

**Pull request merged**